### PR TITLE
New styles for popular docs tiles

### PR DESCRIPTION
**Pull request by tabathadelane**
- dark and light mode hover affects
- ~~and image layout~~ remove image and add link styled button
- used the boop
- responsive!

light mode / desktop
Screenshot 2024-04-12 at 1 25 54 PM

dark mode / small screen
- used Portuguese for responsiveness because it seems to have the most characters
Screenshot 2024-04-12 at 1 26 23 PM

### CODE REVIEW

**Comment by tabathadelane** [comment]
future me: squash your commits

File: src/components/DocTile.js [hunk 1]
```diff
@@ -84,9 +98,55 @@ export const DocTile = ({
...
+          }
+
+          @media screen and (max-width: 1000px) {
+            max-width: unset;
+            padding: 0.8rem;
+            h4 {
+              font-size: 1.25rem;
+            }
+          }
+          @media screen and (max-width: 760px) {
```
Comment (clarkmcadoo) [inline src/components/DocTile.js:137]: this will be taken care of in the < 1000px media query

File: src/pages/index.js [hunk 2]
```diff
@@ -39,46 +39,51 @@ const HomePage = ({ data }) => {
...
+              @media screen and (max-width: 1000px) {
+                grid-template-columns: 1fr;
+                padding: 0 5rem;
+              }
+
+              @media screen and (max-width: 760px) {
+                grid-template-columns: 1fr;
+                padding: 0;
+              }
+              @media screen and (max-width: 400px) {
```
Comment (clarkmcadoo) [inline src/pages/index.js:64]: can we get rid of this?

File: src/pages/index.js [hunk 3]
```diff
@@ -39,46 +39,51 @@ const HomePage = ({ data }) => {
...
+              @media screen and (max-width: 1500px) {
+                align-self: auto;
+              }
+              @media screen and (max-width: 1000px) {
+                grid-template-columns: 1fr;
+                padding: 0 5rem;
+              }
+
+              @media screen and (max-width: 760px) {
+                grid-template-columns: 1fr;
```
Comment (clarkmcadoo) [inline src/pages/index.js:61]: gets taken care of in the < 1000px media query

**Review by sunnyzanchi — APPROVED** [review]

**Pull request merged**